core/generator.ts

```diff
@@ -278,7 +278,10 @@ export class CodeGenerator {
     // Value blocks must return code and order of operations info.
     // Statement blocks must only return code.
     if (!Array.isArray(tuple)) {
-      throw TypeError('Expecting tuple from value block: ' + targetBlock.type);
+      throw TypeError(
+          `Expecting tuple from value block: ${targetBlock.type} See ` +
+          `developers.google.com/blockly/guides/create-custom-blocks/generating-code` +
```

Comment:
Would it be good to include the scheme?  🤔  I know `https://` is assumed, but it might help with clickability in some terminals.